server/oneoff/sync_buffs.py

```diff
@@ -0,0 +1,229 @@
+from app import session_scope
+from app.database.model_spell_translation import ModelSpellTranslation
+from app.database.model_spell_stats import ModelSpellStats
+from app.database.model_item_translation import ModelItemTranslation
+from app.database.model_buff import ModelBuff
+from app.database.model_item import ModelItem
+import oneoff.database_setup
```

Comment:
Prefer importing only the parts we need

> What's the motivation here? I think I prefer the way it was before, since you don't need all of database_setup. Also, I think we may need to move those enums here and then import them from here to database_setup to avoid circular dependencies.